client/app/components/navigation/Bread.tsx

```diff
@@ -96,7 +87,26 @@ export default function Bread({
   if (params && params.formSection) {
     pathNames.pop();
   }
-
+  // 🕹️ Toggle UUID segment to a title segment...
+  // by using title parameter sent from link href
+  // and useState which is maintained between renders of a top-level React component (required for next\back) navigations
+  const searchParams = useSearchParams();
+  const paramTitle = searchParams.get("title") as string;
+  const [crumbTitle, setCrumbTitle] = useState<string>("");
+  useEffect(() => {
+    // Set the title state to rowTitle if it exists
+    if (paramTitle) {
+      setCrumbTitle(paramTitle);
+    }
+  }, [paramTitle]);
```

Comment:
I noticed is that the title takes a second or so to pop in. It seems to take less time to load if we do something like `const [crumbTitle, setCrumbTitle] = useState<string>(paramTitle ?? "");`

What is the purpose of the useEffect here? Does the param not exist sometimes until after the page is loaded?

Reply:
The param will not exist when using page navigation buttons; so, setting up an effect to set the update crumbTitle whenever paramTitle changes and has a value...

**Note:**  changing to `const [crumbTitle, setCrumbTitle] = useState<string>(paramTitle ?? "");` doesn't seem to work on my end?

Reply:
Okay thanks for the explanation that makes sense!